dask-gateway-dev.yaml

```diff
@@ -0,0 +1,58 @@
+# OPTIONAL conda environment file to help get setup with
+# dependencies to build + test dask-gateway locally
+name: dask-gateway-dev
+channels:
+  - conda-forge
+dependencies:
+  # python and pip
+  - python=3.9
+  - pip  
+  
+  # Dependencies to build dask-gateway and dask-gateway-server
+  # NOTE: changes to this block must also be reflected in the following files 
+  # - dask-gateway/setup.py
+  # - dask-gateway-server/setup.py
+  - aiohttp
+  - dask
+  - distributed
+  - cryptography
+  - tornado
+  - traitlets
+  - colorlog
+  - sqlalchemy # see: dask-gateway-server/setup.py extras_require
+  - pykerberos # see: dask-gateway-server/setup.py extras_require 
+  - go=1.17.8  # golang version to  build dask-gateway-server/dask-gateway-proxy
+  - go-cgo     # golang associated dependency to assist with local setup
+  - compilers  # golang associated dependency to assist with local setup
+  
+  # Dependencies required to build
+  # the dask-gateway documentation
+  - skein
+  - sphinx
+  - dask-sphinx-theme
+  
+  # Dependencies used in CI and running tests locally:
+  # dask-gateway/continuous_integration/kubernetes/install.sh
+  - black
+  - colorlog
+  - ipywidgets
+  - jupyterhub
+  - notebook
+  - trustme
+  - flake8
+  - pytest
+  - pytest-asyncio
+  
+  # Useful tooling for local development (kubernetes):
+  # helm and kubectl
+  - kubernetes-helm
+  - kubernetes-client
+  - minikube
+  
+  - pip:
+    - kubernetes-asyncio # see: dask-gateway-server/setup.py extras_require
```

Comment:
```suggestion
```

Reply:
A modern version is available in conda-forge, so I think even though perhaps it wasn't before, it is not being maintained and updated properly on conda-forge.

Reply:
I did not know that kubernetes-asyncio is available on conda-forge, but I'm glad to see that it is. I've updated the environment file accordingly. I may take a look at the feedstock.